utils/Room.js

```diff
@@ -0,0 +1,31 @@
+module.exports = class Room{
+    constructor(){
+        this.listOfPlayers = []
+        this.nextPlayerNumber = 0
```

Comment:
get current player number in

and next player number can be derived from current player